core/src/test/scala/kafka/utils/LoggingTest.scala

```diff
@@ -58,4 +60,20 @@ class LoggingTest extends Logging {
 
     assertEquals(logging.getClass.getName, logging.log.underlying.getName)
   }
+
+  @Test
+  def testLoggerLevelIsResolved(): Unit = {
+    val controller = new Log4jController()
+    val previousLevel = controller.getLogLevel("kafka")
+    try {
+      controller.setLogLevel("kafka", "TRACE")
+      Logger(LoggerFactory.getLogger("kafka.utils.Log4jControllerTest")).trace("test")
```

Comment:
Can we add a comment explaining why we are invoking the logger here?

Reply:
I verified that it works without `com.typesafe.scalalogging.Logger#apply` here, i.e.,  `LoggerFactory.getLogger("kafka.utils.Log4jControllerTest").trace("test")`. Other test cases like `RestServerTest` don't invoke it. It would be better to remove it.